src/Pos/Launcher/Param.hs

```diff
@@ -30,7 +34,17 @@ data BaseParams = BaseParams
     { bpLoggingParams   :: !LoggingParams  -- ^ Logger parameters
     } deriving (Show)
 
--- | Contains algorithm specific & storage parameters for Node.
+-- | Network parameters.
+data NetworkParams = NetworkParams
+    { npDiscovery :: !(Either (Set NodeId) KademliaParams)
+    -- ^ If this value is 'Left', then given peers will be used in static mode.
+    -- Otherwise kademlia with given params will be used.
+    , npTcpAddr   :: !TCP.TCPAddr
```

Comment:
I've found this function in `Network.Transport.TCP`, but it also doesn't explain anything: https://github.com/serokell/network-transport-tcp/blob/a6c04c35f3a1d786bc5e57fd04cf3e2a043179f3/src/Network/Transport/TCP.hs#L570